### PR TITLE
K8s Intrepid release

### PR DESCRIPTION
**Pull request by kaitlynmichael**
Updated pages: 

- https://docs.redis.com/staging/release-k8s-intrepid/kubernetes/release-notes/
- https://docs.redis.com/staging/release-k8s-intrepid/kubernetes/deployment/openshift/_index/
- https://docs.redis.com/staging/release-k8s-intrepid/kubernetes/deployment/openshift/openshift-cli/
- https://docs.redis.com/staging/release-k8s-intrepid/kubernetes/re-clusters/create-aa-database/
- https://docs.redis.com/staging/release-k8s-intrepid/kubernetes/re-databases/ingress_routing_with_istio/
- https://docs.redis.com/staging/release-k8s-intrepid/kubernetes/release-notes/_index/

Jiras:
DOC-994
DOC-1071
DOC-1091
DOC-995
DOC-1092

### CODE REVIEW

File: content/kubernetes/deployment/openshift/openshift-cli.md [hunk 1]
```diff
@@ -24,7 +24,10 @@ cluster with OpenShift.
 
 ## Prerequisites
 
-- [OpenShift cluster](https://docs.openshift.com/container-platform/4.8/installing/index.html) installed at version 4.6 or higher, with at least three nodes (each meeting the [minimum requirements for a development installation]({{< relref "/rs/administering/designing-production/hardware-requirements.md" >}}))
+- [OpenShift cluster](https://docs.openshift.com/container-platform/4.8/installing/index.html) installed, with at least three nodes (each meeting the [minimum requirements for a development installation]({{< relref "/rs/administering/designing-production/hardware-requirements.md" >}}))
+  {{<note>}}
+If you are running an OpenShift 3 version, use the `bundle.yaml` file located in teh `openshift_3_x` folder in the `redis-enterprise-k8s-docs` repo. This folder also contains the custom resource definitions (CRDs) compatible with OpenShift 3.x.
```
Comment (rrelledge) [inline content/kubernetes/deployment/openshift/openshift-cli.md:29]:
```suggestion
If you are running an OpenShift 3 version, use the `bundle.yaml` file located in the `openshift_3_x` folder in the `redis-enterprise-k8s-docs` repo. This folder also contains the custom resource definitions (CRDs) compatible with OpenShift 3.x.  
```

File: content/kubernetes/deployment/openshift/_index.md [hunk 2]
```diff
@@ -47,8 +47,8 @@ To [create a database on an OpenShift 4.x cluster via the OperatorHub]({{< relre
 
 ## For OpenShift via the CLI
 
-To [create a database on an OpenShift 4.x cluster via the CLI]({{< relref "openshift-cli.md" >}}) you need:
+To [create a database on an OpenShift cluster via the CLI]({{< relref "openshift-cli.md" >}}) you need:
```
Comment (rrelledge) [inline content/kubernetes/deployment/openshift/_index.md:50]:
```suggestion
To [create a database on an OpenShift cluster via the CLI]({{< relref "openshift-cli.md" >}}), you need:
```

File: content/kubernetes/re-databases/ingress_routing_with_istio.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+weight: 25
+alwaysOpen: false
+categories: ["Platforms"]
+aliases: [
+    /kubernetes/re-databases/ingress_routing_with_istio.md,
+    /kubernetes/re-databases/ingress_routing_with_istio/,
+]
+---
+
+Redis Enterprise for Kubernetes version `6.2.8-tbd` introduces the ability to use an Istio ingress gateway as an alternative to NGINX or HaProxy ingress controllers.
```
Comment (rrelledge) [inline content/kubernetes/re-databases/ingress_routing_with_istio.md:14]: Is this version "to be decided"?

**Review by rrelledge — APPROVED** [review]
Looks good to me. I just saw a couple of small things.

File: content/kubernetes/re-databases/ingress_routing_with_istio.md (hunk 3 above)
Comment (kaitlynmichael) [inline content/kubernetes/re-databases/ingress_routing_with_istio.md:14]:
```suggestion
Redis Enterprise for Kubernetes version `6.2.8-11` introduces the ability to use an Istio ingress gateway as an alternative to NGINX or HaProxy ingress controllers.
```

**Pull request merged**